Cargo.toml

```diff
@@ -8,7 +8,3 @@ split-debuginfo = "unpacked"
 [profile.release]
 lto = true
 codegen-units = 1
-
-[patch.crates-io]
```

Comment:
@chitralverma we can patch connectorx to use our own fork at https://github.com/roapi/connector-x if it's taking a long time for upstream to merge your PR.

Reply:
Sounds good, can you please sync that fork with the latest upstream

Reply:
Just sent you an invite for write access to the repo ;)